src/main/java/com/softserveinc/ita/pageobjects/ComparisonPage.java

```diff
@@ -0,0 +1,78 @@
+package com.softserveinc.ita.pageobjects;
+
+import com.codeborne.selenide.SelenideElement;
+
+import static com.codeborne.selenide.Condition.appear;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.*;
+import static java.util.stream.Collectors.toList;
+
+public class ComparisonPage extends BasePage<ComparisonPage> {
+
+    private final SelenideElement compareOnlyDifferencesButtonElement = $("button.comparison-settings__toggle");
+
+    public SearchResultPage addAnotherProductToCompare() {
+        $("a.comparison-settings__button")
+                .shouldBe(visible)
+                .click();
+
+        return new SearchResultPage();
+    }
+
+    public ComparisonPage removeProductFromCompare() {
+        $("button.context-menu__toggle").click();
+        $("rz-trash-icon button")
+                .should(appear)
+                .click();
+
+        return this;
+    }
+
+    public String shareComparisonList() {
+        $("button use[href='#icon-link']").click();
+
+        return clipboard().getText();
+    }
+
+    public boolean isOnlyDifferencesButtonDisplayed() {
+        if ($$("ul.products-grid li").size() <= 1) {
```

Comment:
extract into local variable - $$("ul.products-grid li").size()

but I didn't get the logic of the method, what not to use is just return compareOnlyDifferencesButtonElement.isDisplayed();
instead of if statement?

Reply:
Yeah logic is weird :) 
i just dropped the if statemant, thanks for the heads up